server/webapp/WEB-INF/rails.new/app/presenters/api_v1/config/materials/repository_check_connection_representer.rb

```diff
@@ -0,0 +1,44 @@
+##########################################################################
+# Copyright 2016 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+module ApiV1
+  module Config
+    module Materials
+      class RepositoryCheckConnectionRepresenter < ApiV1::BaseRepresenter
```

Comment:
Why is this object different from `PackageRepositoryRepresenter`? Ideally I should be able to submit the same JSON as https://api.go.cd/current/#the-package-repository-config-object to a different endpoint to avoid having to write a slightly different JSONizer.